docs/src/main/asciidoc/spring-cloud-netflix.adoc

```diff
@@ -2021,6 +2021,17 @@ info:
   url: https://github.com/spring-cloud-samples
 ----
 
+Using https health check with self-signed certificates:
```

Comment:
instead of just calling out self-signed certificates I would say

"To enable the health check request to accept all certificates when using HTTPs set `sidecar.accept-all-ssl-certificates` to `true`.

Reply:
Much better! Done, thanks!